string-manipulation-2/src/swap-chars.ts

```diff
@@ -0,0 +1,20 @@
+/* exported swapChars */
+function swapChars(
+  firstIndex: number,
+  secondIndex: number,
+  string: string
+): string {
+  const firstChar = string[firstIndex];
+  const secondChar = string[secondIndex];
+  const stringArray = string.split('');
+
```

Comment:
What are you using this `stringArray` for? You don't need to turn the string into an array here. You should be able to do this with the input string and your output `swapString` variable.